mappings/net/minecraft/util/profiling/jfr/data/GarbageCollectorHeapSummary.mapping

```diff
@@ -0,0 +1,37 @@
+CLASS net/minecraft/unmapped/C_dheagqwo net/minecraft/util/profiling/jfr/data/GarbageCollectorHeapSummary
+	FIELD f_kyfroroz heapUsed J
+	FIELD f_mwobhhhr timestamp Ljava/time/Instant;
+	FIELD f_ugzdgukl timing Lnet/minecraft/unmapped/C_dheagqwo$C_owpxbeuw;
+	METHOD <init> (Ljava/time/Instant;JLnet/minecraft/unmapped/C_dheagqwo$C_owpxbeuw;)V
+		ARG 1 timestamp
+		ARG 2 heapUsed
+		ARG 4 timing
+	METHOD equals (Ljava/lang/Object;)Z
+		ARG 1 other
+	METHOD m_fpydpzpx heapUsed ()J
+	METHOD m_frxmqjrq timing ()Lnet/minecraft/unmapped/C_dheagqwo$C_owpxbeuw;
+	METHOD m_psulbzde createGarbageCollectionData (Ljava/time/Duration;Ljava/util/List;Ljava/time/Duration;I)Lnet/minecraft/unmapped/C_dheagqwo$C_firjsjzh;
+		ARG 0 duration
+		ARG 2 gcTotalDuration
+		ARG 3 totalGCs
+	METHOD m_urhxmwdg timestamp ()Ljava/time/Instant;
+	METHOD m_vxtycupe fromRecordedEvent (Ljdk/jfr/consumer/RecordedEvent;)Lnet/minecraft/unmapped/C_dheagqwo;
+		ARG 0 event
+	CLASS C_firjsjzh GarbageCollectionData
+		FIELD f_dvvsbpwz gcTotalDuration Ljava/time/Duration;
+		FIELD f_hrtwoort allocationRateBytesPerSecond D
+		FIELD f_hwfmfwer totalGCs I
+		FIELD f_icnlqaki duration Ljava/time/Duration;
+		METHOD <init> (Ljava/time/Duration;Ljava/time/Duration;ID)V
+			ARG 1 duration
+			ARG 2 gcTotalDuration
+			ARG 3 totalGCs
+			ARG 4 allocationRateBytesPerSecond
+		METHOD equals (Ljava/lang/Object;)Z
+			ARG 1 other
+		METHOD m_jwnbnzmh getGcOverHeadPercent ()F
+		METHOD m_kkcemshh duration ()Ljava/time/Duration;
+		METHOD m_nrlozewa totalGCs ()I
```

Comment:
Here too, but I'm not sure how that would work with the plural s? Just `totalGcs`?